ownCloud/Messages/MessageGroupCell.swift

```diff
@@ -0,0 +1,248 @@
+					applyAllSwitch?.addTarget(self, action: #selector(applyAllSwitchChanged), for: .primaryActionTriggered)
+
+					applyAllContainer?.addSubview(applyAllSwitch!)
+				}
+
+				if applyAllLabel == nil {
+					applyAllLabel = UILabel()
+					applyAllLabel?.translatesAutoresizingMaskIntoConstraints = false
+					applyAllLabel?.font = UIFont.systemFont(ofSize: UIFont.systemFontSize)
+					applyAllLabel?.text = "Apply to all".localized
+
+					applyAllContainer?.addSubview(applyAllLabel!)
+				}
+
+				if badgeLabel == nil {
+					badgeLabel = RoundedLabel(text: "", style: .token)
+					badgeLabel?.translatesAutoresizingMaskIntoConstraints = false
+				}
+
+				badgeLabel?.isHidden = true
+
+				if showAllLabel == nil {
+					showAllLabel = UIButton(type: .system)
+					showAllLabel?.translatesAutoresizingMaskIntoConstraints = false
+					showAllLabel?.setTitle("Show all", for: .normal)
+					showAllLabel?.addTarget(self, action: #selector(showAllIssues), for: .primaryActionTriggered)
+
+					applyAllContainer?.addSubview(showAllLabel!)
+				}
+
+				if setupLayout, let applyAllContainer = applyAllContainer, let applyAllSwitch = applyAllSwitch, let applyAllLabel = applyAllLabel, let showAllLabel = showAllLabel {
+					NSLayoutConstraint.activate([
+						applyAllSwitch.topAnchor.constraint(equalTo: applyAllContainer.topAnchor, constant: 10),
+						applyAllSwitch.leftAnchor.constraint(equalTo: applyAllContainer.leftAnchor, constant: 20),
+						applyAllSwitch.bottomAnchor.constraint(equalTo: applyAllContainer.bottomAnchor, constant: -10),
+
+						applyAllLabel.leftAnchor.constraint(equalTo: applyAllSwitch.rightAnchor, constant: 10),
+						applyAllLabel.centerYAnchor.constraint(equalTo: applyAllSwitch.centerYAnchor),
+
+						showAllLabel.leftAnchor.constraint(greaterThanOrEqualTo: applyAllLabel.rightAnchor, constant: 20),
+						showAllLabel.rightAnchor.constraint(equalTo: applyAllContainer.rightAnchor, constant: -20),
+						showAllLabel.centerYAnchor.constraint(equalTo: applyAllSwitch.centerYAnchor)
+					])
+				}
+
+				self.applyThemeCollection(theme: Theme.shared, collection: Theme.shared.activeCollection, event: .initial)
+			}
+
+			if let alertView = alertView, let containerView = containerView {
+				self.addSubview(containerView)
+
+				containerView.addSubview(alertView)
+
+				if multiMessage, let applyAllContainer = applyAllContainer, let badgeLabel = badgeLabel {
+					containerView.addSubview(applyAllContainer)
+					containerView.addSubview(badgeLabel)
+				}
+
+				NSLayoutConstraint.activate([
+					containerView.leftAnchor.constraint(equalTo: self.leftAnchor, constant: 20),
+					containerView.rightAnchor.constraint(equalTo: self.rightAnchor, constant: -20),
+					containerView.topAnchor.constraint(equalTo: self.topAnchor, constant: 20),
+					containerView.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: noBottomSpacing ? 0 : -20),
```

Comment:
Please use constants instead of the values in this view

Reply:
Changed the code accordingly. Thanks!